src/App.vue

```diff
@@ -6,13 +6,20 @@
       </router-link>
 
       <h1 class="text-white font-black text-xl">{{ $route.name }}</h1>
+      <div class="flex-grow"></div>
+      <button
```

Comment:
Used the tailwindui button, note that it gets hover effects with `hover:bg-indigo` .   https://tailwindui.com/components/application-ui/elements/buttons